### ISSUE TITLE
Umbraco forms and accessibility

### ISSUE THREAD

**Issue by pa-sr** [umbraco/Umbraco-CMS#3801]
We are in the process of making sure that our site is WCAG2 compliant and one of the errors that we are seeing is in umbraco forms.
The errors being thrown are all the same across all umbraco forms 
1. Input field has no description
2.Text area has no description
3. Label is not connected to a form control

Reproduction
------------

Create a new Form in the Umbraco backoffice, or open an existing form.
Add form controls like drop down and textarea.
Add the form to any Umbraco webpage as a macro (Insert form with Theme)
Load the page / run an accessibility check on the page.

### Expected result

Form controls should have a title attribute to pass accessibility checks

### Actual result

Fails accessibility testing as the following errors are thrown
1. Input field has no description
2.Text area has no description
3. Label is not connected to a form control

**Comment by nul800sebastiaan** [comment]
Hi @pa-sr - while it is possible our current themes are not fully compliant, you are able to build your own themes and that's what we recommend for now.

Documentation about this lives here: 
https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Themes/
